templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+   {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
+      <li><a href="{% url "logout" %}">Logout</a></li>
```

Comment:
The `logout` URL is included, which is part of Django's built-in authentication views. Ensure that the logout view is correctly configured and accessible.